target/linux/qualcommax/ipq807x/base-files/etc/board.d/02_network

```diff
@@ -39,6 +39,8 @@ ipq807x_setup_interfaces()
 	qnap,301w)
 		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4 10g-2" "10g-1"
 		;;
+        linksys,mx4200v1|\
+        linksys,mx4200v2|\
```

Comment:
You should keep it in alphabetical order.

Reply:
The resolution causes a merge conflict

Reply:
> The resolution causes a merge conflict

its been like that before fixing it